packages/kbn-build/src/utils/Package.js

```diff
@@ -0,0 +1,125 @@
+import path from 'path';
+import chalk from 'chalk';
+
+import {
+  runScriptInDir,
+  runScriptInPackageStreaming,
+  installInDir
+} from './npm';
+import { readPackageJson } from './packageJson';
+import { CliError } from './errors';
+
+export class Package {
+  static async fromPath(path) {
+    const pkgJson = await readPackageJson(path);
+    return new Package(pkgJson, path);
+  }
+
+  constructor(packageJson, packageDir) {
+    this._json = packageJson;
+    this.path = packageDir;
+
+    this.packageJsonLocation = path.resolve(this.path, 'package.json');
+    this.nodeModulesLocation = path.resolve(this.path, 'node_modules');
+    this.targetLocation = path.resolve(this.path, 'target');
+  }
+
+  get name() {
+    return this._json.name;
+  }
+
+  get version() {
+    return this._json.version;
+  }
+
+  get dependencies() {
+    return this._json.dependencies || {};
+  }
+
+  get devDependencies() {
+    return this._json.devDependencies || {};
+  }
+
+  get allDependencies() {
```

Comment:
__nit:__ I'd rather make it a method instead of property (`getAllDependencies()`) as it destructures two arrays and creates a new one for every call that isn't obvious for consumer.